hack/spc_build.sh

```diff
@@ -21,11 +21,10 @@ dnf -y update --best --allowerasing
 dnf -y install dnf-plugins-core
 dnf -y copr enable lsm5/container-diff
 dnf -y install autoconf automake btrfs-progs-devel \
-   buildah \
-   container-diff device-mapper-devel golang go-md2man \
+   buildah container-diff device-mapper-devel golang go-md2man \
    git glibc-static gpgme-devel hostname iproute \
    iputils libassuan-devel libseccomp-static make \
-   moby-engine openssl
+   moby-engine podman
```

Comment:
Did you mean to remove openssl?

Reply:
yup, openssl was only added earlier cause I was trying to move registry config inside fedora container.

Reply:
ohh btw, just to be clear, openssl installed by default on the base fedora container, so it's just that I'm just reverting to not installing it here in the first place.

Reply:
:+1: